src/jit/lclvars.cpp

```diff
@@ -4044,12 +4086,14 @@ int Compiler::lvaAssignVirtualFrameOffsetToArg(unsigned lclNum, unsigned argSize
 #if defined(_TARGET_X86_)
         argOffs += sizeof(void *);
 #elif defined(_TARGET_AMD64_)
-#ifndef UNIX_AMD64_ABI
-        // The offset for args need not to be set for System V.
-        // No outgoing args for callees. The spilled args offsets will be set later.
-        varDsc->lvStkOffs = argOffs;
-        argOffs += sizeof(void *);
-#endif // !UNIX_AMD64_ABI
+#ifdef UNIX_AMD64_ABI
```

Comment:
How would this work? Where are you taking care of the discontinuity in the frame for the return address and the ebp save.